EIPS/eip-1014.md

```diff
@@ -0,0 +1,23 @@
+---
+eip: 1014
+title: Skinny CREATE2
+author: Vitalik Buterin (vbuterin)
```

Comment:
GitHub username is recognized when adding the @ in front of it.  Like '@vbuterin'